scout/server/blueprints/variants/templates/variants/str-variants.html

```diff
@@ -37,7 +37,7 @@
     <table class="table table-bordered table-hover">
       <thead>
         <tr>
-          <th style="width:8%" title="Rank position">Rank</th>
+          <th style="width:8%" title="Index">Index</th>
```

Comment:
Good for now! As soon as Stranger 0.5.6 goes into production, we can change it back.

Reply:
Sure!